### PR TITLE
Release 0.30.0

### PR DESCRIPTION
**Pull request by mzabaluev**
Rendered [CHANGELOG](https://github.com/informalsystems/tendermint-rs/blob/release/v0.30.0/CHANGELOG.md)

### CODE REVIEW

**Comment by romac** [comment]
I just merged https://github.com/informalsystems/tendermint-rs/pull/1281 so we need to include those changes in the changelog as well. Sorry for the churn.

**Comment by mzabaluev** [comment]
> I just merged #1281 so we need to include those changes in the changelog as well. Sorry for the churn.

OK, I was going to ask about this myself :) As a breaking change, it makes sense to include in this release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+  An `HttpClient` can be created with a selected mode specified in the new
+  `builder` API, or have the mode changed afterwards (usually after
+  version discovery) by the added `set_compat_mode` method.
+  For `WebSocketClient`, the mode can only be specified at creation via the new
+  `builder` API.
+  ([#1193](https://github.com/informalsystems/tendermint-rs/pull/1193))
+- [`tendermint-abci`] Port ABCI application support to 0.37 Tendermint Core API.
+  No legacy support for 0.34 is provided at the moment.
+  ([#1193](https://github.com/informalsystems/tendermint-rs/pull/1193)).
+- Derive `Hash` on `tendermint::Time`
```
Comment (romac) [inline CHANGELOG.md:55]:
```suggestion
- [`tendermint`] Derive `Hash` on `tendermint::Time`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
+  ([#1278](https://github.com/informalsystems/tendermint-rs/issues/1278))
+- [tendermint-light-client] Show `max_clock_drift` in error raised when header
```
Comment (romac) [inline CHANGELOG.md:57]:
```suggestion
- [`tendermint-light-client`] Show `max_clock_drift` in error raised when header
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,63 @@
...
+  is from the future ([\#1280](https://github.com/informalsystems/tendermint-
+  rs/issues/1280))
```
Comment (romac) [inline CHANGELOG.md:59]:
```suggestion
  is from the future ([\#1280](https://github.com/informalsystems/tendermint-rs/issues/1280))
```

**Pull request merged**